cowprotocol/cowswap
# feat: review TWAP order modal details

# Summary

## Work in progress

Start for implementation for https://github.com/cowprotocol/cowswap/issues/2577

This PR adds
- Price
- Slippage tolerance
- Limit price (incl. fee/slippage) - not correct amount right now and WIP
- Min received (incl. fee) - not sure this one is also correct amount WIP

Missing
- correct amounts and USD values for `limit price` and `min received`
- maybe some styling

![Screenshot 2023-06-02 at 16 02 44](https://github.com/cowprotocol/cowswap/assets/34926005/1218a0a6-dd4e-415e-8b2c-15c28dbd9f4e)

# To test
- Go to safe
- Open this PR as custom app
- Go to advanced orders
- Try to create a transaction so the confirm modal shows
